cmd/opm/registry/cmd.go

```diff
@@ -0,0 +1,26 @@
+package registry
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// NewOpmRegistryCmd returns the appregistry-server command
+func NewOpmRegistryCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "registry",
+		Short: "registry",
```

Comment:
I think you want `Short` to have a slightly longer description that defines what it actually does. Otherwise `--help` won't really tell you anything.

Reply:
addressed